### PR TITLE
Corrected MAU type HWZC_CHWZC -> HWDDC_CHWDDC

### PR DESCRIPTION
**Pull request by evgeniya-engel**

### CODE REVIEW

**Comment by evgeniya-engel** [comment]
@cstirdivant ready for your review. Thank you!

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -2877,6 +2877,15 @@ literals:
 - oil_pressure_sensor:
     flexible_min: 0.0
     flexible_max: 1034214.0
+- oil_tank_temperature_sensor:
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:2880]: I don't think we need to specify `oil_tank` - please use existing `oil_temperature_sensor` and `oil_pressure_sensor`

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 2]
```diff
@@ -2877,6 +2877,15 @@ literals:
...
+    flexible_min: 255.9259259
+    flexible_max: 422.037037
+- oil_tank_pressure_sensor:
+    flexible_min: 0.0
+    flexible_max: 1034214.0
+- oil_tank_differential_pressure_sensor:
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:2886]: remove `tank` from this field name

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -6394,6 +6414,19 @@ TDTM:
   - MONITORING
 
 
+OTKM:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:6417]: rename to OTPM (oil temp pressure monitoring)

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 3 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: fixed

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 2 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/fields/telemetry_fields.yaml]: fixed

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 1 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/fields/telemetry_fields.yaml]: fixed

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**